bundles/org.openhab.binding.mffan/README.md

```diff
@@ -0,0 +1,69 @@
+# MfFan Binding
+
+This binding is used to enable communications between openHAB and "Modern Forms" or "WAC Lighting" WIFI connected, smart, ceiling fans.  
+
+
+## Supported Things
+
+Supported things include: "Modern Forms" and "WAC Lighting" smart ceiling fans.  The binding has been tested with both.
```

Comment:
This are the supported devices, not the supported `Things`. Maybe add these devices as tested or somehting and use the Thing Id listed in the thing-type.xml